code/src/components/LoadingIndicator.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import CircularProgress from '@material-ui/core/CircularProgress'
+import styled from 'styled-components'
+
+const MyCircularProgress = styled(CircularProgress)`
+  margin: 30px 0px;
+`;
```

Comment:
is ; needed in the end because of Material UI?

Reply:
As it is a styled component the semicolon ; is needed :)